packages/babel6/__tests__/babel6.test.js

```diff
@@ -1,15 +1,20 @@
 import test from 'ava'
+import sinon from 'sinon'
 import babel6 from '../index'
 
 test('Babel default options work', (t) => {
   const block = babel6()
 
+  const addLoader = sinon.spy((loaderDef) => config => config)
   const context = {
     fileType: () => '*.js'
   }
 
-  t.deepEqual(block(context), {})
-  t.deepEqual(block.post(context).module.loaders, [
+  t.deepEqual(block(context, { addLoader })({}), {})
+  t.deepEqual(block.post(context, { addLoader })({}), {})
```

Comment:
> I wouldn't even consider it an integration test.

It's not an integration test, right now it's a unit test. But maybe we talk past each other here. It will probably be best to turn it into an integration test, using it in `createConfig()` and look at the resulting config.

Agree? Might do it in another PR, though.